src/components/settings/proposalWorkflow/ProposalWorkflowConnectionsEditor.tsx

```diff
@@ -182,55 +192,64 @@ const ProposalWorkflowConnectionsEditor: React.FC<ProposalWorkflowConnectionsEdi
           isDragDisabled={true}
         >
           {(provided, snapshot) => (
-            <Grid
-              item
-              xs={12}
-              data-cy={`connection_${proposalWorkflowConnection.proposalStatus.shortCode}_${proposalWorkflowConnection.proposalStatus.id}`}
-              ref={provided.innerRef}
-              {...provided.draggableProps}
-              {...provided.dragHandleProps}
-              style={getItemStyle(
-                snapshot.isDragging,
-                provided.draggableProps.style
-              )}
-              className={classes.item}
-              onClick={() => {
-                if (proposalWorkflowConnection.nextProposalStatusId) {
-                  setWorkflowConnection(proposalWorkflowConnection);
-                }
-              }}
-            >
-              {!isDraftStatus(proposalWorkflowConnection) && (
-                <DialogActions className={classes.dialogActions}>
-                  <IconButton
-                    size="small"
-                    className={classes.removeButton}
-                    data-cy="remove-workflow-status-button"
-                    onClick={e => {
-                      e.stopPropagation();
-                      dispatch({
-                        type: EventType.DELETE_WORKFLOW_STATUS_REQUESTED,
-                        payload: {
-                          source: {
-                            index,
-                            droppableId:
-                              proposalWorkflowConnection.droppableGroupId,
+            <>
```

Comment:
Looks pretty good. I'm just curious what wrapping this section with nameless tags does?

Reply:
It is a problem of having multiple root elements and here is a bit more detailed explanation https://reactjs.org/docs/fragments.html